reference/tools.rst

```diff
@@ -67,7 +67,7 @@ If **arch** or **compiler_version** is specified, it will ignore the settings an
 for these parameters.
 
 Parameters:
-    - **settings** (Required): Conanfile settings. Use ``self.settings``.
+    - **conanfile** (Required): Conanfile object. Use ``self`` in a ``conanfile.py``.
```

Comment:
Should this indicate that this is also backward compatible with a ``settings`` argument?

Reply:
Should this change be done in `vcvars_dict()` and `vcvars()` as well?

Reply:
I have opened an issue about this here https://github.com/conan-io/conan/issues/7009